shared-view_components/test/dummy/config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```

Comment:
I think we could stand to remove all of these commented out Ruby files. What do you think?